protocol/agents.json

```diff
@@ -178,6 +178,14 @@
             ],
             "type": "sdk"
         },
+        {
+          "identifier": "ably-python-rest",
+          "versioned": true,
```

Comment:
https://github.com/ably/ably-python/pull/195 introduces `ably-python-rest` as an entry without a version, so this here should be `versioned: false`

Reply:
Then again, I don't actually think `ably-python-rest` is required, since I believe there is only one ably-python package (as opposed to Ruby where you can either install [ably-ruby](https://github.com/ably/ably-ruby) or [ably-ruby-rest](https://github.com/ably/ably-ruby-rest)).

Reply:
https://github.com/ably/ably-common/pull/93/commits/7483964788a44ca91389519a129e430d10925d28

Reply:
https://github.com/ably/ably-python/pull/195/commits/a5c237791504cfbc9e47e57c1e408987d03f84a4

Reply:
nice 👍